services/management/jobs.go

```diff
@@ -105,6 +106,37 @@ func (s *JobsAPIService) StartEchoJob(_ context.Context, req *jobsAPI.StartEchoJ
 	}, nil
 }
 
+func (s *JobsAPIService) StartMySQLBackupJob(ctx context.Context, req *jobsAPI.StartMySQLBackupJobRequest) (*jobsAPI.StartMySQLBackupJobResponse, error) {
+	res, err := s.prepareAgentJob(req.PmmAgentId, models.MySQLBackupJob)
```

Comment:
`prepareServiceJob`, `prepareAgentJob` was introduced because echo job doesn't require any service, it can be executed just by agent. Backups however require some service (MySQL, PG, mongo)

Reply:
I think we will drop `prepareAgentJob` later